libbeat/publisher/client.go

```diff
@@ -76,6 +77,7 @@ func newClient(pub *BeatPublisher) *client {
 		beatMeta: common.MapStr{
 			"name":     pub.name,
 			"hostname": pub.hostname,
+			"version":  version.GetDefaultVersion(),
```

Comment:
@tsg @andrewkroh I rewrote it by passing through the param. Have a look at it again.